### PR TITLE
blog post: TroubleShooting NodeJS Tracing Issues

### PR DESCRIPTION
**Pull request by blumamir**

### CODE REVIEW

**Comment by austinlparker** [comment]
Hey @blumamir we chatted about this in the comms meeting and our decision is that we don't want to host blogs that are summaries + links to another piece. If you would like to submit the full content of the post (with specific vendor calls-to-action stripped out) then we would be happy to host it and give a canonical link back to your blog.

**Review by austinlparker — CHANGES_REQUESTED** [review]
See earlier post in PR thread.

**Comment by blumamir** [comment]
> Hey @blumamir we chatted about this in the comms meeting and our decision is that we don't want to host blogs that are summaries + links to another piece. If you would like to submit the full content of the post (with specific vendor calls-to-action stripped out) then we would be happy to host it and give a canonical link back to your blog.

Thanks @austinlparker
I'll try to find some time soon to update the PR accordingly. In the meantime converting to draft

**Comment by blumamir** [comment]
> Hey @blumamir we chatted about this in the comms meeting and our decision is that we don't want to host blogs that are summaries + links to another piece. If you would like to submit the full content of the post (with specific vendor calls-to-action stripped out) then we would be happy to host it and give a canonical link back to your blog.

@austinlparker  Updated the PR

File: content/en/blog/2022/troubleshooting-nodejs.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
+---
+title: Checklist for TroubleShooting OpenTelemetry NodeJS Tracing Issues
```
Comment (austinlparker) [inline content/en/blog/2022/troubleshooting-nodejs.md:2]:
```suggestion
title: Checklist for Troubleshooting OpenTelemetry Node.JS Tracing Issues
```

**Review by austinlparker — APPROVED** [review]
Some tweaks to the title but otherwise lgtm

File: content/en/blog/2022/troubleshooting-nodejs.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
+---
+title: Checklist for TroublesShooting OpenTelemetry Node.JS Tracing Issues
+linkTitle: TroublesShooting Node.JS Tracing Issues
+date: 2022-02-22
+canonical_url: https://www.aspecto.io/blog/checklist-for-troubleshooting-opentelemetry-nodejs-tracing-issues
+---
+
+[original post][].
```
Comment (chalin) [inline content/en/blog/2022/troubleshooting-nodejs.md:8]: Move this to the end of the post and, ideally turn it into a (short) sentence.

File: content/en/blog/2022/troubleshooting-nodejs.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base';
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+```
+
+### Remote Sampling Decision
+The default sampling behavior (and a very popular one) is that each span inherits the sampling decision from its parent. If the component that invoked your service is **configured not to sample**, then you will **not see spans** from your service as well.
+
+Examples:
```
Comment (chalin) [inline content/en/blog/2022/troubleshooting-nodejs.md:151]: -> Examples include:

**Comment by austinlparker** [comment]
Content seems fine.

**Comment by chalin** [comment]
Merged. I'll make a pass over it after. (I hadn't seen @austinlparker's LGTM from earlier until I confirmed with him just now.)

**Pull request merged**